src/api/account.ts

```diff
@@ -399,14 +403,54 @@ export class Account {
     return getAccountCoinsCount({ aptosConfig: this.config, ...args });
   }
 
+  /**
+   * Queries the acount's APT amount
+   *
+   * @param args.accountAddress The account address we want to get the total count for
+   * @param args.minimumLedgerVersion Optional ledger version to sync up to, before querying
+   * @returns Current amount of account's APT
+   */
+  async getAccountAPTAmount(args: {
+    accountAddress: AccountAddressInput;
+    minimumLedgerVersion?: AnyNumber;
+  }): Promise<number> {
+    await waitForIndexerOnVersion({
+      config: this.config,
+      minimumLedgerVersion: args.minimumLedgerVersion,
+      processorTypes: [ProcessorType.ACCOUNT_TRANSACTION_PROCESSOR],
+    });
+    return getAccountCoinAmount({ aptosConfig: this.config, coinType: "0x1::aptos_coin::AptosCoin", ...args });
```

Comment:
APTOS_COIN is a constant.

```suggestion
    return getAccountCoinAmount({ aptosConfig: this.config, coinType: APTOS_COIN, ...args });
```